### ISSUE TITLE
Add hook to let an app 'burrow in' the viewer

### ISSUE THREAD

**Issue by psvensson** [evothings/evothings-viewer#18]
The idea is to let developers use the viewer as a good-enough custom app to distribute to others. So that if an app calls 'burrow()' for example, it will be saved/stored somewhere and will be opened directly when the viewer is opened, so it looks a bit like it is a stand-alone cordova app built in the usual way.

Perhaps adding some kind of triple-tap menu or swipe motion that let the user bypass the app and get back to the regular viewer mode.

This could use/reuse the work done on the android client cache branch.

**Comment by mikaelkindborg** [comment]
A related idea is to add bookmarks to the Viewer, to make it a browser for IoT apps.

**Comment by psvensson** [comment]
Yes, that's also a good idea. But I think a subset of the functionality would make people very happy since they probably are looking for something that works just like just one, compiled actual app, but without the appstore hassles.

**Comment by mogorman** [comment]
:+1:
